tools/subgraph-example/test/subgraph.ts

```diff
@@ -40,35 +40,36 @@ describe("Subgraph", () => {
       const result = await getData(GRAVATAR_QUERY);
       const gravatars = result.data.gravatars;
 
-      expect(isEqual(gravatars, expected.gravatar.initial)).to.be.true;
+      expect(JSON.stringify(gravatars)).to.equal(JSON.stringify(expected.gravatar.initial));
```

Comment:
Q: why the change here and in the other cases?

Reply:
I had a hard time figuring out what were the totality of the offended changes to the expected vs current.
Once I changed it to this approach I was able to debug much more faster the failing tests, then changed them all for consistency and future work on these tests are more maintanable.

Reply:
Ahh I see. I've encountered a similar issue in the past.
Good fix